src/fhirtypes/ElementDefinition.ts

```diff
@@ -1903,6 +1903,13 @@ export class ElementDefinition {
 
   private checkXhtml(value: string): boolean {
     try {
+      if (this.path.endsWith('div')) {
+        const startsWithDiv = /^\s*<\s*div[\s>]/.test(value);
+        const endsWithDiv = /<\s*\/\s*div\s*>\s*$/.test(value);
```

Comment:
I thought the same thing too, @mint-thompson -- but then I thought that since regexes are already hard enough, that this approach (keeping separate start/end regexes) is a little easier to read, understand, and maintain than combining them together.  What do you think?